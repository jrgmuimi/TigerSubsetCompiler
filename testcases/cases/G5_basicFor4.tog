let
    var happy_loop_ctrl : int
    var sad_loop_ctrl : int
in
    for happy_loop_ctrl:=0 to 10 do 
        printint(happy_loop_ctrl);
        printstring("Happy\n")
    end;
    for sad_loop_ctrl:=20 to 0 do 
        printint(sad_loop_ctrl);
        printstring("Sad\n")
    end

end