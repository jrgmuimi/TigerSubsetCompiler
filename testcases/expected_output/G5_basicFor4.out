MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

0Happy
1Happy
2Happy
3Happy
4Happy
5Happy
6Happy
7Happy
8Happy
9Happy
10Happy

